modules/hal_nordic/nrfx/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+zephyr_library()
+zephyr_library_property(ALLOW_EMPTY TRUE)
+
+# Inherit key properties from the upstream nrfx library.
+zephyr_library_get_current_dir_lib_name(${ZEPHYR_NRF_MODULE_DIR} zephyr_lib_nrfx)
+zephyr_library_include_directories($<TARGET_PROPERTY:${zephyr_lib_nrfx},INCLUDE_DIRECTORIES>)
+zephyr_library_compile_definitions($<TARGET_PROPERTY:${zephyr_lib_nrfx},COMPILE_DEFINITIONS>)
+
+# Definitions copied from Zephyr.
+set(INC_DIR ${NRFX_DIR}/drivers/include)
+set(SRC_DIR ${NRFX_DIR}/drivers/src)
+set(MDK_DIR ${NRFX_DIR}/mdk)
+set(HELPERS_DIR ${NRFX_DIR}/helpers)
+
+macro(mdk_svd_ifdef feature_toggle filename)
```

Comment:
the identical named macro is already defined upstream: https://github.com/zephyrproject-rtos/zephyr/blob/1708574d0a78e687ff832f7c165de3bdaff54cae/modules/hal_nordic/nrfx/CMakeLists.txt#L217

The means it gets defined twice, something which should be done with utmost care as it can result in hard-to-debug issues.